README.md

```diff
@@ -10,6 +11,10 @@ Start by cloning this repository to your local machine.
 git clone https://github.com/pebble-examples/hello-timeline.git
 ```
 
+Enable the timeline for the watchapp by uploading it to the 
```

Comment:
We should tell people that they'll need to generate a new UUID for the app first.